core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1695,6 +1695,10 @@ public String toString() {
           .setDescription("How often the master schedules persistence jobs "
               + "for files written using ASYNC_THROUGH")
           .build();
+  public static final PropertyKey MASTER_PERSISTENCE_BLACKLIST =
+      new Builder(Name.MASTER_PERSISTENCE_BLACKLIST)
+          .setDescription("Patterns to blacklist persist, comma separated, string match, no regex.")
```

Comment:
in description, can you give a few example, like "staging_hive" and etc and why this may help?